### DISCUSSION TITLE
Omission of non-mandatory attributes

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#38] [General]
INSPIRE maintains in its data models some non-mandatory attributes to adapt to the needs of each country. However, this flexibility is sometimes used to avoid the effort of providing all the available data, providing only those that are strictly mandatory. This practice avoids the fine but significantly decreases the data usability. This is the case of e.g. the lack of information for the numberOfDwelling attribute in the Buildings Theme's Datasets.

It is important to highlight that, the effort of providing as many attributes as possible, radically increases the usability of the data provided.

**Comment by naomithiru** [comment]
This issue has been encountered as well in the Tracing use-case. The UWWTD DischargePoints data model provides for more attributes, and thus serving a more useful dataset for a tracing action, as compared to INSPIRE (sewer)Appurtenance datasets.

**Comment by AndersOstmanNovogit** [comment]
This issue is reported in section 4.2.3 in D4.3.

The only solution I see is to use other more suitable data sets. I see Naomi did that.

**Reply by AndersOstmanNovogit** [reply]
Now in D4.3 section 4.3.2
